Portfolio.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE = html>
+<html lang="en">
+<head>
+  	<meta charset="utf-8">
+  	<title>Jenny Allton-Twist</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<link href="Portfoliostyle.css" type="text/css" rel="stylesheet">	
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+	<nav>
+		<ul>
+			<li><a href="#home">Home</a></li>
+			<li><a href="#portfolio">Portfolio</a></li>
+			<li><a href="#aboutme">About Me</a></li>
+			<li><a href="#contact">Contact</a></li>
+		 </ul>
+		 <div>
```

Comment:
is there a reason you have an empty div here?

Reply:
Was going to add a little logo into the nav bar but hadn't yet located a non-cheesy one. Still thinking about it.